--- a/configure.ac
+++ b/configure.ac
@@ -1280,7 +1280,14 @@
 	       pathconf sysconf \
 	       tgetent tigetflag tigetnum tigetstr setupterm initscr resize_term \
-	       getcchar setcchar waddwstr wget_wch win_wch use_default_colors \
+	       getcchar setcchar waddwstr wget_wch win_wch wborder_set use_default_colors \
+	       waddchnstr wadd_wchnstr wgetbkgrnd wbkgrndset \
+	       wattr_get wattr_set \
+	       copywin newpad pnoutrefresh wchgat \
+	       mvwin wresize \
+	       define_key key_defined keybound get_escdelay set_escdelay \
+	       def_prog_mode reset_prog_mode \
+	       init_extended_pair extended_color_content \
 	       nl_langinfo \
 	       erand48 open_memstream \
 	       posix_openpt \
 	       wctomb iconv \
--- a/Doc/Makefile.in
+++ b/Doc/Makefile.in
@@ -70,7 +70,7 @@
 Zsh/mod_regex.yo Zsh/mod_sched.yo Zsh/mod_socket.yo \
 Zsh/mod_stat.yo  Zsh/mod_system.yo Zsh/mod_tcp.yo \
 Zsh/mod_termcap.yo Zsh/mod_terminfo.yo \
-Zsh/mod_watch.yo \
+Zsh/mod_watch.yo Zsh/mod_zdraw.yo \
 Zsh/mod_zftp.yo Zsh/mod_zle.yo Zsh/mod_zleparameter.yo \
 Zsh/mod_zprof.yo Zsh/mod_zpty.yo Zsh/mod_zselect.yo \
 Zsh/mod_zutil.yo
